src/main/java/seedu/address/commons/util/ConfigUtil.java

```diff
@@ -12,6 +12,10 @@
  */
 public class ConfigUtil {
 
+    private ConfigUtil() {
```

Comment:
Great idea to add private default constructors to this and the surrounding classes